Source/Actions/AlertAction.swift

```diff
@@ -24,11 +24,12 @@ public class AlertAction: NSObject {
     - parameter style:   The action's style
     - parameter handler: An optional closure that's called when the user taps on this action
     */
-    public convenience init(title: String?, style: AlertActionStyle, handler: ((AlertAction) -> Void)? = nil)
+    public convenience init(title: String?, style: AlertActionStyle, accessibilityIdentifier: String? = nil, handler: ((AlertAction) -> Void)? = nil)
```

Comment:
To keep the public API clean, let's keep this out of the initializer. Anyone that wants to use this can use the public property instead.